### DISCUSSION TITLE
Supported Spyder version for spyder-modelx 0.7.0

### DISCUSSION THREAD

**Discussion by fumitoh** [fumitoh/modelx#48] [Show and tell]
spyder-modelx 0.7.1 is released on [PyPI](https://pypi.org/project/spyder-modelx/), which now supports v5.0.0

spyder-modelx 0.7.1 should support Spyder v3.2.4 through v5.0.1, but tests are conducted only with the following versions of Spyder.

| Spyder ver. |
|-------------|
| 5.0.0       |
| 4.1.1       |
| 3.3.6       |
| 3.2.5       |

If you happen to use the plugin with untested Spyder and you know it's working, write about it here and let the community know!

**Comment by PatrikMarkku** [comment]
I get spyder-modelx 0.7.1 to work with spyder 5.0.0 and modelx 0.14.0.

The MxConsole, MxExplorer, MxAnalyzer and MxData viewer does not show up under Tools > Panes menus but did magically appear after a while...
